interpreter/binary/decode.ml

```diff
@@ -592,6 +594,13 @@ let memory s =
 let memory_section s =
   section `MemorySection (vec (at memory)) [] s
 
+(* Event section *)
+
+let event s =
+  ignore (vu32 s); var s
```

Comment:
As mentioned above, this should probably invoke an `event_type` decoder function.

Reply:
Done